### DISCUSSION TITLE
aio version 4.3.4 behind apache proxy warnings and error issues

### DISCUSSION THREAD

**Discussion by wsxws** [nextcloud/all-in-one#1965] [Questions] [answered]
Hi all,
after installing aio Version 4.3.4 (channel beta, because latest did not work behind apache proxy) there are some warnings and one error left:

first on the very beginning (after creating docker group):   WARNING: No swap limit support

and then, after ready to handle connections (w/o info messages):
 
```
{"level":"warn","ts":1676125220.5901866,"msg":"Caddyfile input is not formatted; run the 'caddy fmt' command to fix inconsistencies","adapter":"caddyfile","file":"/Caddyfile","line":2}
{"level":"warn","ts":1676125220.5933583,"logger":"http","msg":"automatic HTTP->HTTPS redirects are disabled","server_name":"srv1"}
{"level":"warn","ts":1676125220.593379,"logger":"http","msg":"server is listening only on the HTTP port, so no automatic HTTPS will be applied to this server","server_name":"srv0","http_port":80}
{"level":"warn","ts":1676125220.5935948,"logger":"tls","msg":"YOUR SERVER MAY BE VULNERABLE TO ABUSE: on-demand TLS is enabled, but no protections are in place","docs":"https://caddyserver.com/docs/automatic-https#on-demand-tls"}
{"level":"error","ts":1676125220.5939639,"msg":"unable to create folder for config autosave","dir":"/root/.config/caddy","error":"mkdir /root/.config: permission denied"}
```

hope that helps on developing the new version ;)

BR
wsxws

**Comment by Zoey2936** [comment]
the swap warning is problem of your host system and the warnings can be ignored

**Comment by wsxws** [comment]
i was asked by szaimen on help.nextcloud.com to post this here ;)

**Reply by Zoey2936** [reply]
"caddy ftm" is basically that the caddyfile looks cleaner, so not a problem
as it runs as reverse proxy the redirect warning and tls warnings can be ignored
and the last line only means that caddy cant create a copy of the caddy conf, which is not needed

And with swap, do you use a Raspberry Pi? I have the same error on all pis, I ignore the error for years, since it didn't cause problems for me.

**Comment by wsxws** [comment]
nope, this machine is not even close to a pi.
webserver AMD EPYC 7702P with 64gb ram with 2tb ssd on 2,5gbit lan

but thanks for the infos ;)

**Reply by Zoey2936** [reply]
From the specs, it sounds like a netcup root server (but I would also be wrong). If it is one, which OS do you use and which Kernel/Docker version? Since on my netcup servers I don't have this error. (Even if it is not a netcup server, could you provide docker and kernel version?

**Comment by wsxws** [comment]
right, its netcup ;)

kernel: 5.4.0-73-generic
distri: DISTRIB_RELEASE=20.04
docker-compose: Docker Compose version v2.14.0
docker: Docker version 20.10.22, build 3a2c30b

**Reply by Zoey2936** [reply]
Sorry, no Idea with Debian 11 with Kernel v6 and Docker v23.0.1 it works for me

**Reply by wsxws** [reply]
updated docker to 23, no changes

**Comment by szaimen** [comment, marked as answer]
Hi, the caddy error messages will get removed in a future update with https://github.com/nextcloud/all-in-one/pull/1969

**Reply by szaimen** [reply]
This is now released with v4.4.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
